notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_m1m3_ics_historical_analysis.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<ul><li>I suggest to add a brief explanation on what you observe and what conclusions can be drawn after each set of plots</li><li>A general conclusion would be useful</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
I added something at the beginning of the notebook. Would you have a look, please?